tests/functional/test_install.py

```diff
@@ -19,6 +20,9 @@
 from tests.lib.local_repos import local_checkout
 from tests.lib.path import Path
 
+python2_only = pytest.mark.skipif(not PY2, reason="Python 2 only")
+non_python2_only = pytest.mark.skipif(PY2, reason="Non-Python 2 only")
```

Comment:
I think something like `@skip_if_python2` and `@skip_if_not_python2` might be better names since it parallels `skipif` and `python2_only` isn't so clear out of context (only what?).

Reply:
Great idea, updated!

https://github.com/pypa/pip/pull/6782/commits/2b94acb9d82084abedd6cac5eb5d9e6767ca5dce